spec/controllers/users_controller_spec.rb

```diff
@@ -112,7 +112,20 @@
         end
 
         it 'should save the user' do
-          expect(User.find(assigns(:user))).not_to be_nil
+          expect(User.find(assigns(:user).id)).not_to be_nil
```

Comment:
Yea, that's also a good call. I'll add a shared example for this and use it throughout the context,